File: C/Listas/Lista 2/ex05.c
//Faça um programa que leia A, B e C e calcule a seguinte fórmula: (7*C+5*B) / (2*A*3), armazenando-a na variável Result.

#include <stdio.h>
#include <stdlib.h>

int main(){
    float a,b,c,Result;
    printf("Insira os valores de a, b e c, respectivamente: ");
    scanf("%f%f%f", &a,&b,&c);
    Result=(7*c+5*b)/(2*1*3);

    printf("Fórmula do Cálculo: (7*%.0f+5*%.0f) / (2*%.0f*3) = %.1f",c,b,a,Result);
    return 0;
}

File: Java/extra/Lista 02/src/ex05.java
/* Faça um programa que leia A, B e C e calcule a seguinte fórmula: (7*C+5*B) / 
(2*A*3) */

import java.util.Scanner;

public class ex05 {
    static void calculo(int a, int b, int c) {
        double x = (7 * c + 5 * b) / (2 * a * 3);
        System.out.println("\n" + x);
    }

    public static void main(String[] args) {
        int A, B, C;
        System.out.print("Informe o valor de A: ");
        try (Scanner leitor = new Scanner(System.in)) {
            A = leitor.nextInt();
            System.out.print("Informe o valor de B: ");
            B = leitor.nextInt();
            System.out.print("Informe o valor de C: ");
            C = leitor.nextInt();
        }
        calculo(A, B, C);
    }
}